### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by alandefreitas**

### CODE REVIEW

**Comment by alandefreitas** [comment]
@vinniefalco Can I replicate that in the boost release notes?

File: doc/qbk/release_notes.qbk [hunk 1]
```diff
@@ -13,14 +13,30 @@
...
+* Add http::message_generator
+
+[*Fixes]
+
+* Fix CVE-2018-25032 in zlib streams
+* Fix beast::zlib warnings
+
+[*Improvements]
 
+* Improve documentation
```
Comment (vinniefalco) [inline doc/qbk/release_notes.qbk:39]: remove this

File: doc/qbk/release_notes.qbk [hunk 2]
```diff
@@ -13,14 +13,30 @@
...
+
+[*Features]
+
+* Add buffers_generator
+* Add http::message_generator
+
+[*Fixes]
+
+* Fix CVE-2018-25032 in zlib streams
+* Fix beast::zlib warnings
```
Comment (vinniefalco) [inline doc/qbk/release_notes.qbk:35]: remove this

File: doc/qbk/release_notes.qbk [hunk 3]
```diff
@@ -13,14 +13,30 @@
...
  * `remove_suffix()` and `remove_prefix()` were more lenient than the standard specs; be sure you don't rely on it clamping the argument to valid range
  * `BOOST_NO_CXX11_EXPLICIT_CONVERSION_OPERATORS` no longer suppresses conversions to `std::string`
+* Use `span` from Boost.Core
+* Use `static_string` from Boost.StaticString
+* `serializer::is_done` is `const`
+
+[*Features]
+
+* Add buffers_generator
+* Add http::message_generator
```
Comment (vinniefalco) [inline doc/qbk/release_notes.qbk:30]: if you can link to the related issues it would be good

Comment (sehe) [reply doc/qbk/release_notes.qbk:30]: Should be just the one https://github.com/boostorg/beast/issues/2421

**Pull request closed without merging**